src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -10,271 +10,284 @@
-                             (String) data[4]);
-      Assertion.assertEquals(new UrlBuilder("preview", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[5]);
-      Assertion.assertEquals(new UrlBuilder("sandbox", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[6]);
-      Assertion.assertEquals(new UrlBuilder("sandbox-mercurydev", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[7]);
-      Assertion.assertEquals(new UrlBuilder("dev-dmytror", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[8]);
-      Assertion.assertEquals(new UrlBuilder("staging", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[9]);
+    @Test(groups = {"TestUrlBuilder", "unitTests"})
+    public void urlBuilderTests() {
+        Configuration.setTestValue("qs", "");
+
+        for (Object[] data : TEST_DATA) {
+            System.out.println("Nazwa wikii: " + data[0]);
+            Assertion.assertEquals(new UrlBuilder("prod", (Boolean) data[2])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[3]);
+            Assertion.assertEquals(new UrlBuilder("preview", (Boolean) data[2])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[4]);
+            Assertion.assertEquals(new UrlBuilder("sandbox", (Boolean) data[2])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[5]);
+            Assertion.assertEquals(new UrlBuilder("sandbox-mercurydev", (Boolean) data[2])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[6]);
+            Assertion.assertEquals(new UrlBuilder("dev-dmytror", (Boolean) data[2])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[7]);
+            Assertion.assertEquals(new UrlBuilder("staging", (Boolean) data[2])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[8]);
+        }
+    }
+
+    @Test(groups = {"TestUrlBuilder", "unitTests"})
+    public void urlWithLanguageBuilderTests() {
+        Configuration.setTestValue("qs", "");
+
+        for (Object[] data : TEST_DATA_LANGUAGE) {
+            System.out.println("Nazwa wikii: " + data[0]);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS106)